docs/build/build_zh.md

```diff
@@ -82,6 +82,8 @@ $ sealer build -f Kubefile -t my-kubernetes:v1.19.8 -m cloud .
  -t : 指定构建产出镜像的名称
  -m : 指定构建模式[cloud |container |lite] #默认为lite
  .  : build上下文，指定为当前路径
+ --no-cache:构建过程中不使用缓存
+ --raw-docker:构建时使用的是原生的docker（非sealer定制版docker）
```

Comment:
Should not add this argument, use different base to select use raw docker or not.